<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>الاسم  الاول</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>الهـــــاتف</string>
   </property>
  </widget>
  <widget class="QPushButton" name="labAddFathers">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(223, 221, 220);
color:rgb(0, 0, 0);

font: 18pt &quot;Mohammad Annoktah&quot;;</string>
   </property>
   <property name="text">
    <string>اضافة عضو جديد</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateCouncilDOB">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(25, 21, 59);
color:rgb(255, 255, 255);
font: 12pt &quot;PT Bold Heading&quot;;</string>
   </property>
   <property name="locale">
    <locale language="Arabic" country="Yemen"/>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>d/M/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::LocalTime</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>تاريخ الميلاد</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>المهمـــــــة</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelAddingCouncil">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(194, 64, 0);
color:rgb(255, 255, 255);
font: 15pt &quot;PT Bold Heading&quot;;</string>
   </property>
   <property name="text">
    <string>إلغــــــاء</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveCouncil">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>540</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 170, 127);
color:rgb(255, 255, 255);
font: 15pt &quot;PT Bold Heading&quot;;</string>
   </property>
   <property name="text">
    <string>حفـــــظ</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCouncilFathersName">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>100</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCouncilPhone">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCouncilTask">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCouncilThirName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>اللقب</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>الاسم الثالث</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCouncilSecName">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>150</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;PT Bold Heading&quot;;
color:rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>الاسم الثاني</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtCouncilLName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0)</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtCouncilFathersName</tabstop>
  <tabstop>txtCouncilSecName</tabstop>
  <tabstop>txtCouncilThirName</tabstop>
  <tabstop>txtCouncilLName</tabstop>
  <tabstop>txtCouncilPhone</tabstop>
  <tabstop>dateCouncilDOB</tabstop>
  <tabstop>txtCouncilTask</tabstop>
  <tabstop>btnCancelAddingCouncil</tabstop>
  <tabstop>labAddFathers</tabstop>
  <tabstop>btnSaveCouncil</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
